frontend/src/components/RequireValidation/index.js

```diff
@@ -0,0 +1,29 @@
+import { PropTypes } from "prop-types";
+import { useNavigate } from "react-router-dom";
+import { useEffect, useState } from "react";
+
+const RequireValidation = ({ children }) => {
+  const navigator = useNavigate();
+  const [render, setRender] = useState(null);
+
+  useEffect(() => {
+    if (
+      (children.type.componentName === "WheelPage" &&
```

Comment:
If we change the component names the validation breaks. I don't think that makes sense,  why can't we have separated components for each type of validation?

Reply:
Okay!